src/libs/runtime-utils.js

```diff
@@ -40,6 +40,9 @@ export const defaultLocation = 'US'
 export const defaultComputeZone = 'US-CENTRAL1-A'
 export const defaultComputeRegion = 'US-CENTRAL1'
 
+export const defaultAutoPause = true
```

Comment:
`defaultAutoPause` (and `autopause` by itself) sounds non-descriptive. I liked the previous version (`autopauseEnabled`) which made it clear that it held boolean values. What do you think?